rewrite-java/src/main/java/org/openrewrite/java/style/ImportLayoutStyle.java

```diff
@@ -443,6 +445,45 @@ public void claimImport(Block block, JRightPadded<J.Import> impoort) {
         }
     }
 
+    private static class ImportLayoutInfo {
+        private final Set<JavaType.FullyQualified> classpath;
+        private final List<JRightPadded<J.Import>> originalImports;
+        private @Nullable Set<String> containsClassNameConflict = null;
+
+        ImportLayoutInfo(Set<JavaType.FullyQualified> classpath, List<JRightPadded<J.Import>> originalImports) {

```

Comment:
TODO: java.(x) is not included on the classpath, so classes from java.util.*, java.lang.* etc must be handled separately.